src/components/structures/login/Login.js

```diff
@@ -181,12 +186,20 @@ module.exports = React.createClass({displayName: 'Login',
     componentForStep: function(step) {
         switch (step) {
             case 'm.login.password':
+                // https://matrix.org/jira/browse/SYN-744
+                const loginIncorrect = (
```

Comment:
would it make more sense to have a `loginIncorrect` member of the state (instead of `state.error`)?